src/main/views/webContentEventsCommon.ts

```diff
@@ -7,6 +7,10 @@ import type {Event} from 'electron';
 import type {Logger} from 'common/log';
 import {getLevel} from 'common/log';
 
+import {protocols} from '../../../electron-builder.json';
+
+const scheme = protocols && protocols[0] && protocols[0].schemes && protocols[0].schemes[0];
```

Comment:
For my understanding, what does this do exactly? Not sure if it's worth a helper function and some tests?

Reply:
@enzowritescode It simply extracts the protocol scheme we define for our app (i.e., `mattermost`). Then we check against that and if it doesn't match then it must be custom. It's an exception essentially. 

> Not sure if it's worth a helper function and some tests?

Coverage for that is already at 100% through higher level tests. That said, I moved that definition inside `isCustomProtocol` since it's only used there and it makes sense to be close.